WebJobs.Script.sln

```diff
@@ -448,6 +448,10 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "HttpTrigger-TypeScript", "H
 		sample\HttpTrigger-TypeScript\index.ts = sample\HttpTrigger-TypeScript\index.ts
 	EndProjectSection
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "WebJobs.Script.Scaling", "src\WebJobs.Script.Scaling\WebJobs.Script.Scaling.csproj", "{292A9038-5807-4095-B37F-9A77170D7402}"
```

Comment:
WebJobs.Script.Scaling acts upon the worker status being set for each worker.  How runtime figures out the worker status or where that codes live is entirely runtime implementation.   For instance, each trigger would know how busy it is, so the calculation would be close to trigger implementation.  The host also monitors global resources and can overrule the status, so that codes should live in the host runtime.   IMO.